images/ceph/Makefile

```diff
@@ -66,12 +66,6 @@ do.build:
 	@cp $(MANIFESTS_DIR)/test-data/ceph-status-out $(TEMP)/rook-external/test-data/
 	@$(MAKE) list-image
 
-ifeq ($(INCLUDE_CSV_TEMPLATES),true)
-	@$(MAKE) CSV_TEMPLATE_DIR=$(TEMP) generate-csv-templates
-	@cp -r $(TEMP)/cluster/olm/ceph/templates $(TEMP)/ceph-csv-templates
-else
-	mkdir $(TEMP)/ceph-csv-templates
-endif
```

Comment:
Now that we can generate CSVs separately, let's separate this. When the build also does this and changes files in the same location, it causes conflicts when checking for updates